packages/preview2-shim/lib/nodejs/http.js

```diff
@@ -21,9 +21,12 @@ import {
   registerDispose,
   registerIncomingHttpHandler,
 } from "../io/worker-io.js";
-import { validateHeaderName, validateHeaderValue } from "node:http";
+// import { validateHeaderName, validateHeaderValue } from "node:http";
```

Comment:
```suggestion
import * as http from "node:http";
const { validateHeaderName = () => {}, validateHeaderValue = () => {} } = http;
```

Reply:
That's also a very nice improvement!

I just committed both changes, but it still needs a lot more of additional testing and rework.